src/app/app-design-tool/components/jobs-list/components/job/job.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-list-item>
+  {{job}}
+  <span class="fill-remaining-space"></span>
+  <div align="end" style="display: inline">
```

Comment:
preferably don't use inline style. isn't it possible to achieve this with fxLayout?